tool/planet/main.go

```diff
@@ -289,12 +293,16 @@ func run() error {
 		conf := &agent.Config{
 			Name:        *cagentName,
 			RPCAddrs:    *cagentRPCAddrs,
-			SerfRPCAddr: *cagentSerfRPCAddr,
+			SerfConfig:  serf.Config{Addr: *cagentSerfRPCAddr},
 			MetricsAddr: *cagentMetricsAddr,
 			Cache:       cache,
 			CAFile:      *cagentEtcdCAFile,
 			CertFile:    *cagentEtcdCertFile,
 			KeyFile:     *cagentEtcdKeyFile,
+			TimelineConfig: sqlite.Config{
+				DBPath:            *cagentTimelineDir,
+				RetentionDuration: *cagentRetention,
```

Comment:
How is the default handled? Does it need to be enforced or should there be a reasonable default set from command line?

Reply:
```golang
func (c *Config) CheckAndSetDefaults() error {
	if c.DBPath == "" {
		errors = append(errors, trace.BadParameter("sqlite database path must be provided"))
	}
	if c.Clock == nil {
		c.Clock = clockwork.NewRealClock()
	}
	if c.RetentionDuration == time.Duration(0) {
		c.RetentionDuration = defaultTimelineRentention
	}
}
```
A default duration of 7 days will be set value is not provided. Will update godoc to specify default values.